installers/go-agent/release/agent.sh

```diff
@@ -169,7 +169,10 @@ export LOG_DIR
 eval stringToArgsArray "$AGENT_BOOTSTRAPPER_ARGS"
 AGENT_BOOTSTRAPPER_ARGS=("${_stringToArgs[@]}")
 
-RUN_CMD=("$(autoDetectJavaExecutable)" "-jar" "$AGENT_DIR/agent-bootstrapper.jar" "-serverUrl" "$(autoDetectGoServerUrl)" "${AGENT_BOOTSTRAPPER_ARGS[@]}")
+eval stringToArgsArray "$AGENT_BOOTSTRAPPER_JVM_ARGS"
```

Comment:
`AGENT_BOOTSTRAPPER_ARGS` is meant to be passed to the `main()` of the bootstrapper. `AGENT_BOOTSTRAPPER_JVM_ARGS` is meant to be passed to the JVM.